examples/waspello/tailwind.config.js

```diff
@@ -10,4 +10,4 @@ module.exports = {
     extend: {},
   },
   plugins: [],
-};
+};
```

Comment:
Many of these files are missing newlines at the end. Can prettier take care of that?